packages/css/src/components/radio/radio.scss

```diff
@@ -44,111 +23,181 @@
   gap: var(--ams-radio-gap);
   line-height: var(--ams-radio-line-height);
   outline-offset: var(--ams-radio-outline-offset);
+  position: relative;
+  text-decoration-thickness: var(--ams-radio-text-decoration-thickness);
+  text-underline-offset: var(--ams-radio-text-underline-offset);
+
+  @include text-rendering;
+}
 
-  &:hover {
-    color: var(--ams-radio-hover-color);
-    text-decoration-line: underline;
-    text-decoration-thickness: var(--ams-radio-hover-text-decoration-thickness);
-    text-underline-offset: 0.375rem;
+.ams-radio__icon-container {
+  block-size: var(--ams-radio-icon-container-block-size);
+  display: flex;
+  flex-shrink: 0;
+  inline-size: var(--ams-radio-icon-container-inline-size);
+}
 
-    .ams-radio__circle::after {
-      border-color: var(--ams-radio-circle-hover-border-color);
-    }
+.ams-radio__icon {
+  circle:first-child {
+    stroke: var(--ams-radio-icon-outer-stroke);
+    stroke-width: 0.15rem;
```

Comment:
How do we decide which design choices to store in the SVG and which in CSS?

Reply:
Does the value of `0.15rem` match the design choice for the width of the strokes? I think the designers decided on 1.5 pixels for a canvas of 24 by 24 pixels.

Reply:
> How do we decide which design choices to store in the SVG and which in CSS?

The general idea I used now was SVG for shapes, CSS for style, similar to what we do with HTML and CSS.

> Does the value of 0.15rem match the design choice for the width of the strokes?

Good catch, I'm not sure where `0.15` came from. In the designs [here](https://www.figma.com/design/9IGm6IdPUYizBNGsUnueBd/branch/UzLI9D01t8POzFIpg46CBe/Amsterdam-Design-System?node-id=2927-29187&node-type=canvas&t=wlbVJASvfrUBoIXd-0) stroke width is 2px, so it should be `0.125rem` I think?